### DISCUSSION TITLE
API to get and describe tables

### DISCUSSION THREAD

**Discussion by logikaljay** [supabase/supabase#1641] [Questions] [answered]
Hey team,

I have checked how app.supabase.io does it, it calls an API `GET /api/database/{project}/tables`, however this endpoint I believe requires a session cookie.

I have tried calling it in postman with the service key to no avail.

Is there a supported method to get a list of tables, and also describe a table (or tables as per the above endpoint) using the service key?

**Comment by inian** [comment, marked as answer]
Hi @logikaljay, thats an internal API. It does some authentication and in turn call [postgrest-meta](https://github.com/supabase/postgres-meta). You can deploy postgrest-meta yourself (it is a docker container) and point it to your database. More docs on postgres-meta [here](https://github.com/supabase/postgres-meta)
